doc/nrf/ug_thread_tools.rst

```diff
@@ -79,50 +79,21 @@ As neither the Linux-based PC nor the Raspberry Pi have such radio capability, y
 
 To program the nRF device with the RCP application, complete the following steps:
 
-#. Clone the OpenThread nRF528xx platform repository into the current directory:
-
-   .. code-block:: console
-
-      git clone --recursive https://github.com/openthread/ot-nrf528xx.git
-
-#. Enter the :file:`ot-nrf528xx` directory:
-
-   .. code-block:: console
-
-      cd ot-nrf528xx
-
-#. Install the OpenThread dependencies:
-
-   .. code-block:: console
-
-      ./script/bootstrap
-
-#. Build the RCP example for the hardware platform and the transport of your choice:
+#. Build the :ref:`ot_coprocessor_sample` sample for the hardware platform and the transport of your choice:
 
    .. tabs::
 
       .. tab:: nRF52840 Dongle (USB transport)
 
          .. code-block:: console
 
-            rm -rf build
-            script/build nrf52840 USB_trans -DOT_BOOTLOADER=USB -DOT_THREAD_VERSION=1.2
+            west build -b nrf52840dongle_nrf52840 -- -DOVERLAY_CONFIG="overlay-rcp.conf ../common/overlay-thread_1_2_ftd.conf overlay-usb.conf"
```

Comment:
nit: I think that `overlay-usb.conf` could be automatically included when building for `nrf52840dongle_nrf52840` but this is not related to this PR.